client/components/zendesk-chat-widget/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+
+interface Props {
+	chatId: string | false;
+}
+
+const ZendeskChat = ( { chatId }: Props ) => {
```

Comment:
Nitpick: maybe let's call this `chatKey` to be consistent with `zendeskPresalesChatKey` and `zendesk_presales_chat_key` from the config files. It can also be confused with `sessionId` or some other identifier of the chat (as named in Happy Chat), not the chat widget.